tools/perf/lib/benchmark/runner/fio.py

```diff
@@ -112,7 +112,6 @@ def __init__(self, benchmark, config: dict, idfile: str) -> 'FioRunner':
         self.__direct_write_to_pmem = \
```

Comment:
There should be no changes in this file in this commit. This commit only adds unit tests.